sunpy/tests/__init__.py

```diff
@@ -17,8 +17,8 @@
 
 
 def main(modulename='', coverage=False, cov_report=False,
-         online=False, offline=True, remote_data=False, figure=False, verbose=False,
-         parallel=0, args=None):
+         online=False, offline=True, remote_data=False, figure=False, figure_only=False,
+         verbose=False, parallel=0, args=None):
```

Comment:
Would need the new keyword being added to the documentation string below.

Reply:
Yeah I'll add it in my next commit. Thanks.